### ISSUE TITLE
Drop Platform "WindowsPhone"?

### ISSUE THREAD

**Issue by DDReaper** [MonoGame/MonoGame#5318]
Like Ouya, Windows Phone 7 is probably about as dead as it's going to get.
We can't publish WP7 titles anymore and for those still updating WP7 titles, they can use the existing release as it's highly unlikely it's going to be updated.

**Comment by KonajuGames** [comment]
We never supported Windows Phone 7.  Line XBLIG, XNA was the only supported path for publishing games.  The WindowsPhone platform is Windows Phone 8.

**Comment by DDReaper** [comment]
My mistake @KonajuGames however the statement remains the same.  Should the MG framework drop the older WP support and leave it with the current release as what is being done with Ouya?

**Comment by Syncaidius** [comment]
Having had a MG game on WP8 for some years, I'm of the opinion that WP8 has reached the point where there are so few users left on it that making any games for it is no longer profitable for game devs or worth the effort from MG team/contributors.

I think it would be better for both the MG team and developers to shift remaining effort on WP8 to UWP. The latter covers more hardware platforms, is newer and more likely to generate an income for developers. Unlike WP8, UWP also has massive potential-user pool thanks to Win10, so its likely to stay around longer and worth considering the shift of support.

If anyone does still need/want WP8, they can still aquire it from a previous MG release/commit.

**Comment by Danthekilla** [comment]
Isn't 95%+ of the windows phone 8 code the same as the windows 8 codepath?

And about 85% is the same as windows 10, so I don't know how much maintenance it really requires.

Perhaps someone with more knowledge can weigh in.

**Comment by DDReaper** [comment]
WP8.1 shares more similarities with Win8.1 than it does with wp8.0.

Wp8.0 has a fair amount of unique code due to it's architecture, also there are a lot of specific exclusions for WindowsPhone (just look in the build files for net)

Primary aim is to drop the template / nuget & project generation. Any remaining shared code will stay.

Although as pointed out in this original pass, the build server will need updating too

**Comment by Halofreak1990** [comment]
As any Windows Phone that runs WP8 can upgrade to 8.1, I think dropping WP8 won't be too much of an issue. It becomes slightly problematic when dropping 8.1, since not all 8.x phones are eligible for the upgrade to Windows 10 Mobile.

**Comment by tomspilman** [comment]
> Primary aim is to drop the template / nuget & project generation. Any remaining shared code will stay.

Agreed.

> the build server will need updating too

It is just a setting that needs to be changed... it can be done after the PR removing things is done.  Just leave the targets in the Nant file but make them do nothing.

**Comment by DDReaper** [comment]
Closed with #5336

**Comment by hach-que** [comment]
I think WindowsPhone was also the only platform that needed those weird XNA redirects in order to compile properly.  Unless there's any complaints, I'll put it on the Protobuild roadmap to remove WindowsPhone as a supported platform there as well.

**Comment by tomspilman** [comment]
> I think WindowsPhone was also the only platform that needed those weird XNA redirects in order to compile properly. 

Actually I think it was for Windows Phone 8.1 specifically which we still support.  @DDReaper can you verify this?

**Comment by DDReaper** [comment]
The backwards fixes were only needed for WindowsPhone 8.0. WindowsPhone8.1 didn't use them as far as I was aware.
```
  <**Platform Type="WindowsPhone"**>
    <Reference Include="Microsoft.Xna.Framework">
      <Alias Platform="WindowsPhone">MicrosoftXnaFramework</Alias>
    </Reference>
  </Platform>
```
